ethpm/package.py

```diff
@@ -98,39 +96,29 @@ def from_file(cls, file_path_or_obj: str, w3: Web3) -> "Package":
         return cls(package_data, w3)
 
     @classmethod
-    def from_ipfs(cls, ipfs_uri: str, w3: Web3) -> "Package":
+    def from_uri(cls, uri: str, w3: Web3) -> "Package":
         """
-        Instantiate and return a Package object from an IPFS URI pointing to a manifest.
+        Return a Package object instantiated by a manifest located at a content-addressed URI.
+        URI schemes supported:
+            - IPFS          `ipfs://Qm...`
+            - HTTP          `https://raw.githubusercontent.com/repo/path.json#hash`
+            - Registry      `ercXXX://registry.eth/greeter?version=1.0.0`
         """
-        ipfs_backend = get_ipfs_backend()
-        if ipfs_backend.can_handle_uri(ipfs_uri):
-            raw_package_data = ipfs_backend.fetch_uri_contents(ipfs_uri)
-            package_data = json.loads(to_text(raw_package_data))
+        good_backends = get_backends_for_uri(uri)
```

Comment:
Maybe just call this `backends` since there isn't another set of `bad_backends` in scope here.